wemake_python_styleguide/cli/commands/explain/module_loader.py

```diff
@@ -0,0 +1,47 @@
+import importlib
+from collections.abc import Collection
+from contextlib import suppress
+from pathlib import Path
+from types import ModuleType
+from typing import Final
+
+_VIOLATION_MODULE_BASE: Final = 'wemake_python_styleguide.violations'
+
+
+def get_violation_submodules() -> Collection[ModuleType]:
+    """Get all possible violation submodules."""
+    return _safely_get_all_submodules(_VIOLATION_MODULE_BASE)
+
+
+def _safely_get_all_submodules(module_name: str) -> Collection[ModuleType]:
+    """Get all submodules of given module. Ignore missing."""
+    submodule_names = _get_all_possible_submodule_names(module_name)
+    modules = []
+    for submodule_name in submodule_names:
+        # just in case if there are some bad module paths
+        # (which generally should not happen)
+        with suppress(ModuleNotFoundError):
```

Comment:
```suggestion
```

Let's see if there are any cases like this really. It better to show the full exc in the beginning.